avocado/utils/asset.py

```diff
@@ -50,11 +50,11 @@ class Asset:
     Try to fetch/verify an asset file from multiple locations.
     """
 
-    def __init__(self, name, asset_hash=None, algorithm=None, locations=None,
-                 cache_dirs=None, expire=None, metadata=None):
+    def __init__(self, name=None, asset_hash=None, algorithm=None,
+                 locations=None, cache_dirs=None, expire=None, metadata=None):
```

Comment:
Hi @beraldoleal I think that this commit is not related to this PR and was already merged in #4427